test/Ocelot.Benchmarks/Ocelot.Benchmarks.csproj

```diff
@@ -25,6 +25,7 @@
 
   <ItemGroup>
     <PackageReference Include="BenchmarkDotNet" Version="0.13.1" />
+    <PackageReference Include="Serilog.AspNetCore" Version="7.0.0" />
```

Comment:
Perfect! We need to discuss that!
Also, let Raynald know that you've tested Serilog, not Microsoft default logger. 🙃

Reply:
@RaynaldM Benchmark done with Serilog, but unit tests with Microsoft default logger

Reply:
@ggnaegi Gui, you did great job. Thanks!

But, if you test another user scenarios which are very useful for your team and your projects, it is totally incorrect delivery in this open source project. We need not replace or introduce new features hiding or replacing old ones. I mean, we must keep scenarios of those developers who like and use Microsoft logger of the box. Clear? And, I'm absolutely fine to keep this Serilog reference because you've prepared great Serilog logger user scenario. 🥳 
It seems, you need a help, because I see this PR requires a bit more development.
Do you need a help for Microsoft loggers scenarios?